src/Seq.App.Slack/SlackApi.cs

```diff
@@ -1,24 +1,48 @@
-﻿using System.Net.Http;
+﻿using System.Net;
+using System.Net.Http;
 using System.Text;
 using Newtonsoft.Json;
 
 namespace Seq.App.Slack
 {
-    static class SlackApi
+    class SlackApi
     {
-        private static readonly HttpClient HttpClient = new HttpClient();
+        private HttpClient httpClient;
```

Comment:
`readonly`?

Reply:
Nit: the codebase uses `_underscore` property name prefixes.

Reply:
fixed, made readonly and used underscore